sdcm/cluster_aws.py

```diff
@@ -436,6 +462,10 @@ def private_ip_address(self):
 
         return self._instance.private_ip_address
 
+    @property
+    def ipv6_ip_address(self):
+        return self._instance.network_interfaces[0].ipv6_addresses[0]["Ipv6Address"]
```

Comment:
Does this ipv6 address work between the multiple AWS regions?

Reply:
"We currently do not support Elastic IP addresses for IPv6."

That's from AWS documentation.

For supporting it, we need to buy an public IPv6 address space and assign it to a subnet. 

So no we don't support it now

Reply:
So currently we can't use ipv6 for multiple dc testing?

Reply:
I assume it won't work. It's not the goal of this PR. 

Maybe VPC peering would enable us to achieve that, but again that's beyond the scope of this PR

Reply:
AWS doesn't use the `Unique Local` range, so it can be routed on the internet.
It seems we only added ipv6 CIRD for vpc of eu-west-1, vpc of us-east-1 doesn't support ipv6, so I failed to test it.

Reply:
> I assume it won't work. It's not the goal of this PR.

Sure.

> Maybe VPC peering would enable us to achieve that, but again that's beyond the scope of this PR

Reply:
I think that the Ipv6 we have is global, so If we enable ipv6 in another region we should be able to reach this network using IPv6.
So in theory we can test multi-DC with IPv6.
@fruch, I don't see how elastic IP for IPv6 is related here...

Reply:
@roydahan is right IPv6 is always public (global). So we will be able to reach it from anywhere in the world (still need to configure VPC in relevant regions). We don't need Elastic IPv6.

Reply:
@roydahan  Do open the task for that?

Reply:
Yes, please.

Reply:
Done. 
https://trello.com/c/mkR7X3lV/1356-test-multi-dc-with-ipv6